[WIP] Feature/quick review of hosting patches

helmo opened the pull request:
I'd like to 'abuse' this branch continued review.

Most commits will have a Drupal.org issue number included, please leave any comments there.

See also:
- https://github.com/aegir-project/hostmaster/pull/5
- https://github.com/aegir-project/provision/pull/11

jonpugh commented:
@helmo how about just name the branch `review` or `patch-review` then?  It's hardly a feature ;)

jonpugh commented:
I restarted the build

jonpugh commented:
It's the  `git log --oneline -n1` command. It keeps the terminal open.  I'm rusty with bash. Make sure the git log command only outputs, not follows.

jonpugh commented:
https://github.com/aegir-project/hosting/commits/7.x-3.x/.travis.yml

See the red dot? beware, unless we lock the branch, we can push failing code.

jonpugh commented:
I guess i'm not sure!  sorry, very tired, typing with one hand

helmo commented:
On 25-10-16 17:24, Jon Pugh wrote:

> @helmo https://github.com/helmo how about just name the branch
> |review| or |patch-review| then? It's hardly a feature ;)

True ;)

> I guess i'm not sure! sorry, very tired, typing with one hand

Hope your more  rested now ....

 The "git log --oneline -n1" command is not something that follows, it
just outputs oneline... as I understand it :)

But anyway, I removed it and it now turned green :)

helmo commented:
WTF ... the commit after the one where I introduced that debug line passed.

https://travis-ci.org/aegir-project/hosting/builds/169495658

But after cherry-picking two commits to 7.x-3.x it failed the tests...
and  now removed the debug line there and it passed again ....

jonpugh committed:
Issue #2824731: On site insert, if platform_node data is passed, create platform node.

helmo committed:
Code cleanup

helmo committed:
Point to the companion branch in provision

helmo committed:
Add Views exposed filters to the Task queue page

jonpugh committed:
Issue #2823827: Be more specific in task titles for tasks that are the same across objects: Verify Site, Verify Platform, Verify Server.

jonpugh committed:
Issue #2754069: Use verify instead of Install for first task.

Commit:
Issue #922252 by Neograph734, anarcat: Prevent clients from hijacking each others subdomains

Commit:
Issue #2212149 by Jon Pugh: Improve task queue AJAX refresh

jonpugh committed:
Issue #2849795: Check a variable before running queued tasks, allowing users to pause the queue daemon.

jonpugh committed:
Issue #2849795: Change hosting queue daemon init message based on paused state.  Ensure global $conf is called so variable reloading works.

jonpugh committed:
Issue #2849795: Add a checkbox to the hosting queue daemon settings page for pausing the queue.

helmo committed:
Merge remote-tracking branch 'origin/2849795-pause-queued' into feature/quick-review

Commit:
Issue #2854498 by memtkmcc: Platform path is not sanitized as expected

Commit:
Issue #2854738 by kfolsom: "Allowed Users" Form Not Shown on Client Node Page
